traits-stubs/traits-stubs/api.pyi

```diff
@@ -18,9 +27,30 @@ from .traits import (
     Trait as Trait
 )
 
+from .ctrait import CTrait as CTrait
```

Comment:
Q: Do we want the whole module to follow the same order as is currently in `api.py` (related to your comment about order in `.has_traits` imports)?  If so, `.ctrait` import is before `.traits` imports.

Reply:
That would be nice, one day, and I was trying to move towards that, but doing that in this PR would be quite disruptive; I'd rather leave that sort of cleanup to a separate PR that does nothing more than move lines around (for ease of reviewing).

Reply:
fine by me. resolving.